app/models/agenda-item-treatment.js

```diff
@@ -23,4 +23,5 @@ export default Model.extend({
     });
   }),
   publicationFlows: hasMany('publication-flows'),
+  signFlows: hasMany(),
```

Comment:
~Ik vermoed dat als je geen target type specifieert de naam van de relation gebruikt wordt?~ [Gevonden in de docs](https://api.emberjs.com/ember-data/3.27/functions/@ember-data%2Fmodel/belongsTo)

Ik wist niet dat dat kon eerlijk gezegd. Wat denken jullie @ValenberghsSven @MikiDi ? Beter het type expliciet specifieren zoals we tot nu toe altijd gedaan hebben of is het zo ook goed?

Reply:
Om de code consistent te houden, zou ik opteren om het type wel expliciet mee te geven. (dit zou anders de eerste keer zijn)
`signFlows: hasMany(sign-flow),`
sidenot: klopt de lijn erboven `publicationFlows: hasMany('publication-flows'),`? Ik zou zeggen dat je een enkelvoudig model in de haakjes zet `publicationFlows: hasMany('publication-flow'),`